lxd/device/disk.go

```diff
@@ -114,6 +116,10 @@ func (d *disk) validateConfig() error {
 		return fmt.Errorf("The recursive option is only supported for additional bind-mounted paths")
 	}
 
+	if !strings.HasPrefix(d.config["source"], "ceph") && (d.config["ceph.cluster_name"] != "" || d.config["ceph.user_name"] != "") {
```

Comment:
Please check for both `ceph:` and `cephfs:` prefix (including colon to avoid potential false positives).

Reply:
This one still needs fixing

Reply:
Still needs updating ^